Userland/Libraries/LibJS/Runtime/ArrayBufferPrototype.cpp

```diff
@@ -128,6 +130,28 @@ JS_DEFINE_NATIVE_FUNCTION(ArrayBufferPrototype::slice)
     return new_array_buffer_object;
 }
 
+// 25.1.5.5 ArrayBuffer.prototype.transfer ( [ newLength ] ), https://tc39.es/proposal-arraybuffer-transfer/#sec-arraybuffer.prototype.transfer
+JS_DEFINE_NATIVE_FUNCTION(ArrayBufferPrototype::transfer)
+{
+    // 1. Let O be the this value.
+    auto array_buffer_object = TRY(typed_this_value(vm));
+
+    // 2. Return ? ArrayBufferCopyAndDetach(O, newLength, preserve-resizability).
```

Comment:
```suggestion
    // 2. Return ? ArrayBufferCopyAndDetach(O, newLength, preserve-resizability).
```

(not sure what OS/browser you're using, but we commonly replace the NBSPs from the spec with regular spaces, some do that automatically when copying)